stubs/Flask-SocketIO/flask_socketio/__init__.pyi

```diff
@@ -0,0 +1,97 @@
+from collections.abc import Callable
+from logging import Logger
+from threading import Thread
+from typing import Any
+from typing_extensions import NotRequired, ParamSpec, TypeAlias, TypedDict, Unpack
+
+from flask import Flask
+from flask.testing import FlaskClient
+from socketio.base_manager import BaseManager  # type: ignore[import]  # pyright: ignore[reportMissingImports]
```

Comment:
This line is now passing mypy and pyright due to the `type: ignore`s, but, as you can see, it's still causing issues with pytype. For most other stubs where we have this problem, we do this instead:

```py
# Actually socketio.base_manager.BaseManager,
# but socketio isn't py.typed and typeshed has no stubs for it
BaseManager: TypeAlias = Any
```